src/main/menu_main.cpp

```diff
@@ -421,7 +460,7 @@ bool mainMenuUpdate()
                     MenuMode = MENU_BATTLE_MODE;
                     menuPlayersNum = 2;
                     menuBattleMode = true;
-#ifdef __EMSCRIPTEN__
+#if defined(__EMSCRIPTEN__) || defined(__3DS__)
```

Comment:
I would simplify this:
- at the top of the file, add next:
```cpp
#if defined(__EMSCRIPTEN__) || defined(__3DS__)
#define MENU_NO_THREADED_LOAD
#endif
```

- in the code, simply use `#ifdef MENU_NO_THREADED_LOAD` and `#ifndef MENU_NO_THREADED_LOAD`